core/server/common/src/main/java/alluxio/master/journal/raft/transport/CopycatGrpcBaseConnection.java

```diff
@@ -44,10 +44,10 @@
 import java.util.function.Function;
 
 /**
- * Copycat transport {@link Connection} implementation that uses Alluxio gRPC.
+ * Base copycat transport {@link Connection} implementation that uses Alluxio gRPC.
  */
-public class CopycatGrpcConnection implements Connection, StreamObserver<CopycatMessage> {
-  private static final Logger LOG = LoggerFactory.getLogger(CopycatGrpcConnection.class);
+public class CopycatGrpcBaseConnection implements Connection, StreamObserver<CopycatMessage> {
```

Comment:
Should this be an abstract class now?

Reply:
It's functional by itself. I can mark it abstract for restricting its use.

Reply:
All connections should be either client or server right? If there is a use case for the base connection its fine. I would prefer to be more restrictive if possible.